src/configuration/services/saas.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Magento Services
+redirect_from:
+  - https://devdocs.magento.com/recommendations/configure.html
+---
+
+Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Services]({% link configuration/services.md %}) >  Magento Services
+
+Magento allows you to link your Magento store to a SaaS service through the Admin UI. By configuring and managing SaaS Environments, you can connect to services such as Product Recommendations.
+
+When you create a [Magento account]({% link magento/magento-account.md %}), identified by a MageID, you can generate a Magento API key to connect to the SaaS services within Magento. To connect to Magento Commerce SaaS services, the API key needs to be generated by the license-holder. Then the API key can be passed the systems integrator or development team, who can then manage the SaaS projects and environments on behalf of the license-holder.
+
+Each MageID can have one SaaS project and within that SaaS project you can have one or more SaaS environments depending on the type of Magento license you have:
```

Comment:
depending on whether or not you have a license - or something similar 
reason: open source is not a license "type", in that case you just don't have a license
the options below may not need to change, up to you, but I would list them as
- Magento Commerce
- Magento Open Source